media-fonts/encodings/encodings-1.0.5-r1.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# font eclass is inherited directly, since this package is a special case that
+# would greatly complicate the fonts logic of xorg-3
+inherit font xorg-3
+
+DESCRIPTION="X.Org font encodings"
+
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~sparc-solaris ~x64-solaris ~x86-solaris"
+DOCS=( ChangeLog README.md )
+
+BDEPEND="x11-apps/mkfontscale
+	>=media-fonts/font-util-1.1.1-r1"
+
+src_install() {
+	xorg-3_src_install
+	einstalldocs
+}
```

Comment:
The diff here looks like

```diff
-pkg_postinst() {
-	xorg-2_pkg_postinst
-	font_pkg_postinst
-}
-
-pkg_postrm() {
-	xorg-2_pkg_postrm
-	font_pkg_postrm
+src_install() {
+	xorg-3_src_install
+	einstalldocs
 }
```

Do we really not want the `pkg_*` functions?

Dropping the `DOCS` assignment, we can also drop `src_install`.